helm/jaeger-all-in-one/templates/statefulset.yaml

```diff
@@ -85,10 +85,18 @@ spec:
               name: jaeger-volume
           {{- end }}
           livenessProbe:
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
             httpGet:
               path: {{ .Values.healthCheckUrl | quote }}
               port: http-admin
           readinessProbe:
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
```

Comment:
These should be `.Values.readinessProbe.` not `.Values.livenessProbe`

Reply:
Good catch, thanks 🙏 Fixed